torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11140,6 +11144,7 @@ def ref_pairwise_distance(input1, input2):
            dtypesIfCUDA=floating_types_and(torch.float16, *[torch.bfloat16] if CUDA11OrLater else []),
            sample_inputs_func=partial(sample_inputs_conv2d),
            gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
+           supports_forward_ad=True,
```

Comment:
There is no test for conv3d?

Reply:
Nope, good catch. I'm wasn't too worried since we don't have dim-specific code and we do test for ConvTranspose3d. Should we be?

Reply:
But that can be done in another PR ofc